components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -58,19 +58,27 @@ class BrowserToolbar @JvmOverloads constructor(
     /**
      * Set/Get whether a site security icon (usually a lock or globe icon) should be next to the URL.
      */
-    var displaySiteSecurityIcon: Boolean
+    var siteSecurityIcon: Boolean
```

Comment:
My first thought to this variable is that it will be to it give me access to siteSecurityIcon not is visibility. What about including the visibility on the name to clarify its purpose?
something like`isSiteSecurityIconVisible`.

Reply:
I'm also confused about this change. Why is it needed? :)

Reply:
Thanks for the catch, this wasn't intended to be committed!